T/TermInfoDB/build_tarballs.jl

```diff
@@ -0,0 +1,71 @@
+using BinaryBuilder
+
+# This is based on https://www.freshports.org/misc/terminfo-db/
+name = "TermInfoDB"
+version = v"2023.12.9"
+
+v = string(version.major, lpad(version.minor, 2, '0'), lpad(version.patch, 2, '0'))
+sources = [
+    FileSource("https://invisible-island.net/archives/ncurses/current/terminfo-$v.src.gz",
+               "2debcf2fd689988d44558bcd8a26a104b96542ffc9540f19e2586b3aeecd1c79"),
+    DirectorySource("./bundled"),
+]
+
+script = raw"""
+cd ${WORKSPACE}/srcdir/
+gunzip terminfo-*
+mkdir -p "${prefix}/share/terminfo"
+tic -sx -o "${prefix}/share/terminfo" ./terminfo-*
+
+# When ignoring case, these files are duplicates of others. We'll remove them to ensure
+# they don't cause trouble on case-insensitive filesystems
+DUPS=(
```

Comment:
There are single-letter files which are still problematic: https://buildkite.com/julialang/yggdrasil/builds/12154#0190ff34-b5bd-41c5-b1e4-3b1109eb8df8/673-903

Reply:
I see the warning from the auditor but when I download the artifact and look at the contents, I don't see the things it's complaining about 🤔